### PR TITLE
Add clinical tracks to oncoprinter

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

File: src/pages/resultsView/mutualExclusivity/MutualExclusivityTab.tsx [hunk 1]
```diff
@@ -76,11 +76,21 @@ export default class MutualExclusivityTab extends React.Component<IMutualExclusi
         this.significantPairsFilter = !this.significantPairsFilter;
     }
 
+    @computed get error() {
```
Comment (alisman) [inline src/pages/resultsView/mutualExclusivity/MutualExclusivityTab.tsx]: can this have a more descriptive name?  e.g. XYZBrandErrorText

File: src/pages/staticPages/tools/oncoprinter/Oncoprinter.tsx [hunk 2]
```diff
@@ -229,6 +235,11 @@ export default class Oncoprinter extends React.Component<IOncoprinterProps, {}>
     private oncoprintRef(oncoprint:OncoprintJS) {
         this.oncoprint = oncoprint;
 
+        onMobxPromise(this.props.store.alteredSampleIds,
```
Comment (alisman) [inline src/pages/staticPages/tools/oncoprinter/Oncoprinter.tsx]: seems like a kinda weird place to put this no?

File: src/pages/staticPages/tools/oncoprinter/OncoprinterClinicalUtils.ts [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        ret.push({
+            clinicalAttributeName: match[1],
+            datatype: datatype as ClinicalTrackDataType
+        });
+    }
+
+    return ret;
+}
+
+export function parseClinicalInput(input:string):{ status:"complete", result:{ headers:AttributeSpec[], data:OncoprinterClinicalInputLine[]}, error:undefined }|{ status:"error", result:undefined, error:string } {
```
Comment (alisman) [inline src/pages/staticPages/tools/oncoprinter/OncoprinterClinicalUtils.ts]: maybe a comment that the response type mimics mobxpromise?

Comment (adamabeshouse) [reply src/pages/staticPages/tools/oncoprinter/OncoprinterClinicalUtils.ts]: sure, but just for the record it's not meant to be used in "await", its just a convenient form for giving back the result

File: src/pages/staticPages/tools/oncoprinter/Oncoprinter.tsx (hunk 2 above)
Comment (adamabeshouse) [reply src/pages/staticPages/tools/oncoprinter/Oncoprinter.tsx]: sure that makes sense

File: src/pages/resultsView/mutualExclusivity/MutualExclusivityTab.tsx (hunk 1 above)
Comment (adamabeshouse) [reply src/pages/resultsView/mutualExclusivity/MutualExclusivityTab.tsx]: sure

**Pull request merged**